bsp/imx/apalis-imx8.scc

```diff
@@ -46,4 +46,7 @@ include features/tee/tee.scc
 
 include cfg/usb-mass-storage.scc
 
+# Exclude x86/ACPI features
+include cfg/non-x86.cfg
```

Comment:
While I would prefer fixing the fragments the other way around (moving stuff that is specific to x86 to the respective x86 boards/bsps), this is better than what we have now and easier to maintain, so fine with it.

Reply:
Me either :)  We could do it a bit later when having some free time :)